firmware/quadruna/VC/src/app/states/app_initState.c

```diff
@@ -12,6 +23,14 @@ static void initStateRunOnTick1Hz(void)
 static void initStateRunOnTick100Hz(void)
 {
     app_allStates_runOnTick100Hz();
+
+    bool is_inverter_on_or_drive_state =
+        app_canRx_BMS_State_get() == BMS_INVERTER_ON_STATE || app_canRx_BMS_State_get() == BMS_DRIVE_STATE;
+
+    if (is_inverter_on_or_drive_state)
+    {
+        app_stateMachine_setNextState(app_driveState_get());
```

Comment:
You may need another state here so the state transitions for the VC can go from init -> inverter_on -> drive. Where you would have the PDM `is_inverter_on_or_drive_state` check in init and the DCM to drive state checks in inverter_on state

Reply:
done!